src/record_syscall.cc

```diff
@@ -4357,6 +4357,41 @@ static void check_privileged_exe(RecordTask* t) {
   }
 }
 
+#define AT_ENTRY_X86_64 152
+#define AT_ENTRY_X86 84
```

Comment:
We can't hardcode the offsets like this. Instead we have to walk the auxv vector looking for an entry whose key is `AT_ENTRY`.